rootfs/rootfs/etc/rc.d/vmtoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/local/bin/vmtoolsd --background /var/run/vmtoolsd.pid
```

Comment:
Thanks :heart:

If it is indeed a good citizen, let's at least add a comment noting as much
so that we know where the assumption was encoded if we manage to find a
situation it's not a good citizen in later. :+1: